deepcell_tracking/isbi_utils.py

```diff
@@ -329,21 +330,16 @@ def classify_divisions(G_gt, G_res):
     }
 
 
-def benchmark_division_performance(trk_gt, trk_res, path_gt, path_res):
+def benchmark_division_performance(trk_gt, trk_res):
```

Comment:
Again, maybe we can leave them but set default to None and warn if used.

It looks like we can safely ignore the input if the user provides it.